web/client/product/app.jsx

```diff
@@ -32,7 +33,8 @@ const startApp = () => {
 
     const initialActions = [
         () => loadMaps(ConfigUtils.getDefaults().geoStoreUrl, ConfigUtils.getDefaults().initialMapFilter || "*"),
-        loadVersion
+        loadVersion,
+        verifySession
```

Comment:
This is ok, but I'd like to have it connected directly to the plugin, as an epic. So I don't have to bring it in all installations.
Is it possible?